src/validation/fieldValidators/index.ts

```diff
@@ -0,0 +1,5 @@
+export { name } from './name'
+export { notKeyword } from './notKeyword'
+export { numeric } from './numeric'
+export { numericPositive } from './numericPositive'
+export { required } from './required'
```

Comment:
here you can export const FieldsValidator = {name,....}

Reply:
ok